infra/terraform/environments/dev/main.tf

```diff
@@ -253,6 +253,17 @@ module "service" {
     repository = data.aws_ecr_repository.this["cli"].repository_url
 
     task_iam_role_statements = local.task_iam_role_statements
+    eventbridge_iam_policy_statements = {
+      batch = {
+        effect = "Allow"
+        actions = [
+          "batch:SubmitJob"
+        ]
+        resources = [
+          "arn:aws:batch:eu-west-1:054614622558:job-definition/vol-app-dev-*"
+        ]
+      }
+    }
```

Comment:
Would it be better to put this in the service module?

We're in control of that set of resources in the service module so will not change between environments, and we could build up the wildcard ARN: `arn:aws:batch:eu-west-1:054614622558:job-definition/vol-app-${var.environment}-*`.

This will remove duplication across the environment files.

Reply:
or even an explicit list?

```hcl
{
    effect = "Allow"
    actions = [
      "batch:SubmitJob"
    ]
    resources = [ for job in var.batch.jobs : module.batch.job_definitions[job.name].arn ]
}
```